src/app/components/left-menu/left-menu.component.scss

```diff
@@ -1,58 +1,46 @@
+@import '../../../styles/variables.scss';
```

Comment:
Could be replaced by `@import 'styles/variables.scss` 
if `baseUrl` is set to src in tsconfig

Reply:
Will be fixed in next PR